### DISCUSSION TITLE
Welcome to WebApp Discussions!

### DISCUSSION THREAD

**Discussion by OtabekgithubJava** [OtabekgithubJava/WebApp#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by OtabekgithubJava** [comment]
@akmalxoja

**Comment by akmalxoja** [comment]
Nme gap

**Comment by akmalxoja** [comment]
@IsmoilDotnet

**Comment by OtabekgithubJava** [comment]
@IsmoilDotnet

**Comment by IsmoilDotnet** [comment]
Hi guys

**Reply by IsmoilDotnet** [reply]
🌭
